### PR TITLE
DRM Wizard

### PR DESCRIPTION
**Pull request by elken**
- Added a wizard to find DRM games (uplay, origin & steam atm)
- Shrunk Library ctor
- Few minor changes elsewhere

NOTE: Until I find out where they're stored, user-configured launch options don't carry over.

EDIT: Ok I just found the file, hope is yet.

### CODE REVIEW

File: Source/Main.cpp [hunk 1]
```diff
@@ -1,24 +1,32 @@
 #include "Init.h"
 #include "BuddyList.h"
 
+#include <QApplication>
+#include <QFile>
+#include <QDebug>
+#include <iostream>
+#include "Wizard.h"
```
Comment (lolzballs) [inline Source/Main.cpp]: This should be moved up to the other local includes.

File: Source/Wizard.cpp [hunk 2]
```diff
@@ -0,0 +1,775 @@
+#include <QtWidgets>
+#include <iostream>
+#include "Wizard.h"
```
Comment (lolzballs) [inline Source/Wizard.cpp]: These includes should follow the style documented.

File: Source/Wizard.h [hunk 3]
```diff
@@ -0,0 +1,120 @@
+#pragma once
+
+#include <QtWidgets>
+#include <vector>
+#include "Database.h"
```
Comment (lolzballs) [inline Source/Wizard.h]: Same thing

**Comment by lolzballs** [comment]
Any particular reason for not including documentation?

**Comment by elken** [comment]
I just wanted it done & looked over. I'll add docs before merge.

**Comment by elken** [comment]
@lolzballs @Jtramey Docs are in, check it out and when one of you two signs it off I'll merge

**Comment by elken** [comment]
Merged #95 into this one

**Comment by awslattery** [comment]
Nice work guys.

File: Source/MainPanel.cpp [hunk 4]
```diff
@@ -26,7 +28,7 @@ MainPanel::MainPanel(QWidget* parent)
 */
 void MainPanel::init()
 {
-    p = new QSettings("palette.ini", QSettings::IniFormat);
+    p = new QSettings(QSettings::IniFormat, QSettings::UserScope, "Project Ascension", "palette");
```
Comment (elken) [inline Source/MainPanel.cpp:31]: @Hal9007

**Pull request merged**